Source/CronosPlayUnreal/Public/DefiWalletCoreActor.h

```diff
@@ -30,6 +30,27 @@ destroyErc1155();
 */
 
 // callback
+// general
+DECLARE_DYNAMIC_DELEGATE_ThreeParams(FGeneralCronosCallBoolDelegate, bool,
+                                     Output, bool, Success, FString,
+                                     OutputMesssage);
+
+DECLARE_DYNAMIC_DELEGATE_ThreeParams(FGeneralCronosCallInt64Delegate, int64,
+                                     Output, bool, Success, FString,
+                                     OutputMesssage);
+
+DECLARE_DYNAMIC_DELEGATE_ThreeParams(FGeneralCronosCallInt32Delegate, int64,
```

Comment:
the name says `FGeneralCronosCallInt32Delegate`, but the argument seems to be int64 -- not sure if intended, or duplicate (enough to have `FGeneralCronosCallInt64Delegate`?) ...

Reply:
it's intended, but i'll change